config/runtime.exs

```diff
@@ -87,6 +89,5 @@ if config_env() == :prod do
     secret_key_base: secret_key_base
 
   config :sentry,
-    dsn: System.get_env("SENTRY_DSN"),
-    environment_name: System.get_env("ENVIRONMENT")
```

Comment:
I think this config value is still needed for sentry's use on the backend. So it can still reference `System.get_env("ENVIRONMENT")`. Sorry for being unclear. We should have both `config :sentry, evnironment_name` AND `config, :orbit, :environment`, one for sentry (backend) to use, and one for everything else to use.

Reply:
Ah! gotcha. I should have tested in sandbox like I was doing yesterday. Will fix-